src/services/chat/prepopulated.js

```diff
@@ -41,17 +41,17 @@ module.exports = async (id, users) => {
             better_channel_member: [
               {
                 user: {
-                  ...ownUser
+                  ...ownUser.defaulValues
                 },
-                user_id: ownUser?.id,
+                user_id: ownUser?.user_id,
                 is_anonymous: false,
                 updated_at: new Date().toISOString()
               },
               {
                 user: {
-                  ...user
+                  ...user.defaulValues
                 },
-                user_id: ownUser?.id,
+                user_id: user?.user_id,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `ownUser?.id` to `ownUser?.user_id` and `user?.user_id` seems to be correct assuming that the `user_id` property is the correct one to use. However, without more context about the structure of these objects, it's hard to say for sure. Please ensure that `user_id` is the correct property to use.



<!-- This is an auto-generated comment by OSS CodeRabbit -->